src/app/base/components/node/OverviewCard/_index.scss

```diff
@@ -13,92 +13,96 @@
     }
 
     .overview-card__cpu {
-      @include pseudo-border(left);
+      @media only screen and (min-width: ($breakpoint-small + 1)){
+        @include pseudo-border(left);
+      };
       grid-area: cpu;
       padding: $spv--large 0 0 0;
-      &::after {
-        top: $spv--large;
-      }
     }
 
     .overview-card__memory {
-      @include pseudo-border(left);
+      @media only screen and (min-width: ($breakpoint-large + 1)){
+        @include pseudo-border(left);
+      };
       grid-area: memory;
       padding: $spv--large 0 0 0;
-      &::after {
-        top: $spv--large;
-      }
     }
 
     .overview-card__storage {
-      @include pseudo-border(left);
+      @media only screen and (min-width: ($breakpoint-small + 1)){
+        @include pseudo-border(left);
+      };
       grid-area: storage;
       padding: $spv--large $sph--large 0 0;
-      &::after {
-        top: $spv--large;
-      }
     }
 
     .overview-card__test-warning {
       grid-area: test-warning;
       padding: $spv--large 0 $spv--large $sph--large;
     }
 
+    .overview-card__test-text {
+      margin-bottom: 0.1rem;
+      padding-top: calc(0.4rem - 1px);
+    }
+
+    .overview-card__storage-test-text {
+      margin-bottom: 0.1rem;
+      padding-top: calc(0.4rem - 1px);
+    }
+
+    .overview-card__memory-test-text {
+      margin-bottom: 0.1rem;
+      padding-top: calc(0.4rem - 1px);
+    }
+
+    .overview-card__cpu-test-text {
+      margin-bottom: 0.1rem;
+      padding-top: calc(0.4rem - 1px);
+    }
+
     .overview-card__cpu-tests {
       @include pseudo-border(left);
       grid-area: cpu-tests;
       padding: $spv--large 0;
-      &::after {
-        bottom: $spv--large;
-      }
     }
 
     .overview-card__memory-tests {
       @include pseudo-border(left);
       grid-area: memory-tests;
       padding: $spv--large 0;
-      &::after {
-        bottom: $spv--large;
-      }
     }
 
     .overview-card__storage-tests {
       @include pseudo-border(left);
       grid-area: storage-tests;
       padding: $spv--large $sph--large $spv--large 0;
-      &::after {
-        bottom: $spv--large;
-      }
     }
 
     .overview-card__details {
       @include pseudo-border(top);
-      display: grid;
+      display: flex;
+      flex: 1 0 auto;
+      justify-content: space-between;
+      gap: $sph--large;
       grid-area: details;
-      grid-template-rows: auto;
-      padding: $spv--large $sph--large;
-
-      &.for-controller {
-        grid-template-columns: minmax(0, 1fr) minmax(0, 1fr) minmax(0, 1fr) minmax(
-            0,
-            4fr
-          );
-      }
-
-      &.for-machine {
-        grid-template-columns: repeat(7, minmax(0, 1fr));
-      }
+      padding: $spv--large 0;
+      margin: 0 1rem 0 1rem ;
```

Comment:
This looks like it should be using spacing variables and also for all the other 1rems in this file.